specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+    },
+    "/devices": {
+      "get": {
+        "operationId": "Devices_List",
+        "summary": "Get the list of devices in an application",
+        "x-ms-examples": {
+          "List devices": {
+            "$ref": "./examples/devices_list.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceCollection"
+            }
+          }
+        }
+      }
+    },
+    "/devices/{device_id}": {
+      "get": {
+        "operationId": "Devices_Get",
+        "summary": "Get a device by ID",
+        "description": "Get details about an existing device by device ID.",
+        "x-ms-examples": {
+          "Get device by Id": {
+            "$ref": "./examples/devices_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Device"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Devices_Create",
+        "summary": "Create or update a device",
+        "description": "Create a new device.",
+        "x-ms-examples": {
+          "Create device": {
+            "$ref": "./examples/devices_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device body.",
+            "schema": {
+              "$ref": "#/definitions/Device"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Device"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Devices_Update",
+        "summary": "Update a device via patch",
+        "description": "Update an existing device by ID.",
+        "x-ms-examples": {
+          "Update device": {
+            "$ref": "./examples/devices_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device patch body.",
+            "schema": {
+              "type": "object"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Device"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Devices_Remove",
```

Comment:
Http verb is delete and summary/description says delete. Why is the operation Id "remove"?

Reply:
@drwill-ms, almost forget this part, I caught the comments for our previous PR, we use remove instead delete is because generated nodejs lib as delete is js keyword.

Reply:
Doesn't seem to be a problem in [ADT swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2020-10-31/digitaltwins.json#L1221) and we have a JS SDK generated for that.

Reply:
we did that test for last PR review,  the delete method generated in nodejs SDK with method called "deletemethod", added a "method" suffix on generated method name. with "remove" we can keep it.

Reply:
I don't know why it worked for us in ADT, but it doesn't work here. I've pinged the scripting team to weigh in.

Reply:
Sounds good, if it can keep the .delete() as  generated method, we should use "Delete".

Reply:
Yeah, I don't think that was an issue for us. For example, [this operation in the ADT swagger](https://github.com/Azure/azure-rest-api-specs/blob/4f4419354e66e087d884a420d8f486e10d71cce5/specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2020-10-31/digitaltwins.json#L223-L253) has "delete" and there is no suffix added to the [corresponding function that gets generated](https://github.com/Azure/azure-sdk-for-js/blob/36436fdee129dfd51196f9bdee579b48e0558e7c/sdk/digitaltwins/digital-twins-core/src/generated/operations/digitalTwinModels.ts#L144-L169). 

@zolvarga might have something to say here.

Reply:
Get a generated section from nodejs, ADT seems generated typescript lib.
    
/**
 * @summary Delete an API token.
 *
 * @param {​​​​​​​string}​​​​​​​ tokenId Unique ID for the API token.
 *
 * @param {​​​​​​​object}​​​​​​​ [options] Optional Parameters.
 *
 * @param {​​​​​​​object}​​​​​​​ [options.customHeaders] Headers that will be added to the
 * request
 *
 * @param {​​​​​​​function}​​​​​​​ callback - The callback.
 *
 * @returns {​​​​​​​function}​​​​​​​ callback(err, result, request, response)
 *
 *                      {​​​​​​​Error}​​​​​​​  err        - The Error object if an error occurred, null otherwise.
 *
 *                      {​​​​​​​null}​​​​​​​ [result]   - The deserialized result object if an error did not occur.
 *
 *                      {​​​​​​​object}​​​​​​​ [request]  - The HTTP Request object if an error did not occur.
 *
 *                      {​​​​​​​stream}​​​​​​​ [response] - The HTTP Response stream if an error did not occur.
 */
function _deleteMethod(tokenId, options, callback) {​​​​​​​
   /* jshint validthis: true */
  let client = this.client;
  if(!callback && typeof options === 'function') {​​​​​​​
    callback = options;
    options = null;
  }​​​​​​​
  if (!callback) {​​​​​​​
    throw new Error('callback cannot be null.');
  }​​​​​​​
  // Validate
  try {​​​​​​​
    if (this.client.baseDomain === null || this.client.baseDomain === undefined || typeof this.client.baseDomain.valueOf() !== 'string') {​​​​​​​
      throw new Error('this.client.baseDomain cannot be null or undefined and it must be of type string.');
    }​​​​​​​
    if (this.client.subdomain === null || this.client.subdomain === undefined || typeof this.client.subdomain.valueOf() !== 'string') {​​​​​​​
      throw new Error('this.client.subdomain cannot be null or undefined and it must be of type string.');
    }​​​​​​​
    if (tokenId === null || tokenId === undefined || typeof tokenId.valueOf() !== 'string') {​​​​​​​
      throw new Error('tokenId cannot be null or undefined and it must be of type string.');
    }​​​​​​​
    if (this.client.acceptLanguage !== null && this.client.acceptLanguage !== undefined && typeof this.client.acceptLanguage.valueOf() !== 'string') {​​​​​​​
      throw new Error('this.client.acceptLanguage must be of type string.');
    }​​​​​​​
  }​​​​​​​ catch (error) {​​​​​​​
    return callback(error);
  }​​​​​​​


  // Construct URL
  let baseUrl = this.client.baseUri;
  let requestUrl = baseUrl + (baseUrl.endsWith('/') ? '' : '/') + 'apiTokens/{​​​​​​​tokenId}​​​​​​​';
  requestUrl = requestUrl.replace('{​​​​​​​baseDomain}​​​​​​​', this.client.baseDomain);
  requestUrl = requestUrl.replace('{​​​​​​​subdomain}​​​​​​​', encodeURIComponent(this.client.subdomain));
  requestUrl = requestUrl.replace('{​​​​​​​tokenId}​​​​​​​', encodeURIComponent(tokenId));
  let queryParameters = [];
  queryParameters.push('api-version=' + encodeURIComponent(this.client.apiVersion));
  if (queryParameters.length > 0) {​​​​​​​
    requestUrl += '?' + queryParameters.join('&');
  }​​​​​​​


  // Create HTTP transport objects
  let httpRequest = new WebResource();
  httpRequest.method = 'DELETE';
  httpRequest.url = requestUrl;
  httpRequest.headers = {​​​​​​​}​​​​​​​;
  // Set Headers
  httpRequest.headers['Content-Type'] = 'application/json; charset=utf-8';
  if (this.client.generateClientRequestId) {​​​​​​​
      httpRequest.headers['x-ms-client-request-id'] = msRestAzure.generateUuid();
  }​​​​​​​
  if (this.client.acceptLanguage !== undefined && this.client.acceptLanguage !== null) {​​​​​​​
    httpRequest.headers['accept-language'] = this.client.acceptLanguage;
  }​​​​​​​
  if(options) {​​​​​​​
    for(let headerName in options['customHeaders']) {​​​​​​​
      if (options['customHeaders'].hasOwnProperty(headerName)) {​​​​​​​
        httpRequest.headers[headerName] = options['customHeaders'][headerName];
      }​​​​​​​
    }​​​​​​​
  }​​​​​​​
  httpRequest.body = null;
  // Send Request
  return client.pipeline(httpRequest, (err, response, responseBody) => {​​​​​​​
    if (err) {​​​​​​​
      return callback(err);
    }​​​​​​​
    let statusCode = response.statusCode;
    if (statusCode !== 204) {​​​​​​​
      let error = new Error(responseBody);
      error.statusCode = response.statusCode;
      error.request = msRest.stripRequest(httpRequest);
      error.response = msRest.stripResponse(response);
      if (responseBody === '') responseBody = null;
      let parsedErrorResponse;
      try {​​​​​​​
        parsedErrorResponse = JSON.parse(responseBody);
        if (parsedErrorResponse) {​​​​​​​
          if (parsedErrorResponse.error) parsedErrorResponse = parsedErrorResponse.error;
          if (parsedErrorResponse.code) error.code = parsedErrorResponse.code;
          if (parsedErrorResponse.message) error.message = parsedErrorResponse.message;
        }​​​​​​​
        if (parsedErrorResponse !== null && parsedErrorResponse !== undefined) {​​​​​​​
          let resultMapper = new client.models['CloudError']().mapper();
          error.body = client.deserialize(resultMapper, parsedErrorResponse, 'error.body');
        }​​​​​​​
      }​​​​​​​ catch (defaultError) {​​​​​​​
        error.message = `Error "${​​​​​​​defaultError.message}​​​​​​​" occurred in deserializing the responseBody ` +
                         `- "${​​​​​​​responseBody}​​​​​​​" for the default response.`;
        return callback(error);
      }​​​​​​​
      return callback(error);
    }​​​​​​​
    // Create Result
    let result = null;
    if (responseBody === '') responseBody = null;


    return callback(null, result, httpRequest, response);
  }​​​​​​​);
}​​​​​​​

Reply:
we will keep "remove" in current cases.